bus-mapping/src/evm/opcodes/return_revert.rs

```diff
@@ -6,12 +6,12 @@ use crate::{
     },
     evm::opcodes::ExecStep,
     operation::{AccountField, AccountOp, CallContextField},
-    state_db::CodeDB,
     Error,
 };
 use eth_types::{
     bytecode::BytecodeElement,
     evm_types::{memory::MemoryWordRange, OpcodeId},
+    state_db::CodeDB,
```

Comment:
next time i may suggest write 'pub use eth_types::state_db::CodeDB' in bus-mapping/statedb.rs.  So less changes.